canonical/ubuntu-advantage-client
# postinst cleanups + remove root_mode

## Test Steps

LMK if you disagree with the postinst changes.
LMK if you disagree with the new util function for is_root (it's intended to be a bit more readable)
Integration tests should still pass.
Manually run the Pro fips tests.
Manually run the new notice code in postinst to make sure it works as expected.

## Checklist

 - [x] I have updated or added any documentation accordingly (these are all "under-the-hood" changes - no docs necessary)

## Does this PR require extra reviews?

 - [x] No (? unless you think we need extra reviews because of the changes to postinst)
